### DISCUSSION TITLE
Introducing Amazing Subscribers

### DISCUSSION THREAD

**Discussion by vnetonline** [oqtane/oqtane.framework#4308] [Show and tell]
I have just released my first commercial module using [Amazing Oqtane Studio](https://learnoqtane.com/modules/amazing-oqtane-studio) called [Amazing Subscribers](https://learnoqtane.com/modules/amazing-subscribers) 

I have am dog flooding Amazing Subscriber module on the [amazingoqtanestudio.com](https://learnoqtane.com/modules/amazing-oqtane-studio) website and also on [LearnOqtane.com ](https://learnoqtane.com/)

Check it out by visiting [Amazing Subscribers](https://learnoqtane.com/modules/amazing-subscribers)

![screenshot-amazing-subscriber](https://github.com/oqtane/oqtane.framework/assets/15242923/9e56fa46-ef0b-4a1c-89bb-6d9f707729a0)

![screenshot-amazing-subscriber-manage-subscribers](https://github.com/oqtane/oqtane.framework/assets/15242923/8107bd74-3641-427c-aa9c-369b3c0c211f)

![screenshot-amazing-subscriber-send-notifications](https://github.com/oqtane/oqtane.framework/assets/15242923/efe7a770-09d9-43a9-ae88-0945329e91d6)

**Comment by sbwalker** [comment]
@vnetonline it would be useful to explain how this is different from the existing Blog module (https://github.com/oqtane/oqtane.blogs)? It appears from the screen shots that it provides very similar functionality.

**Comment by vnetonline** [comment]
@sbwalker  The Amazing difference is that it has a newsletter functionality independent of any module and is built into the module and it can be used on any page. In addition you can specify an Audience Name .... it also sends the admin of the site an email when someone subscribes ...... the blog module doesn't do that and is tightly coupled with the blog entries.

The future roadmap for this module is to integrate with MailChimp to export audience name via API.

Check out [https://learnoqtane.com/modules/amazing-subscribers](https://learnoqtane.com/modules/amazing-subscribers) for a full explanation of what this module can do.

It was also built using the [Amazing Oqatne Studio](https://amazingoqtanestudio.com) so took only couple of hours.
